bitnami/argo-workflows/values.yaml

```diff
@@ -0,0 +1,1025 @@
+    enabled: true
+    runAsUser: 1001
+
+  ## Server container port
+  ## @param controller.ports.container Argo Workflows controller container port
+  ports:
+    container: 6060
+
+  ## Controller persistence parameters. It can also be connected to an existing postgresql. Check
+  ## @param controller.persistence.enabled Enabled controller persistence
+  ## @param controller.persistence.connectionPool.maxIdleConns Msximun number of postgresql connections
+  ## @param controller.persistence.connectionPool.maxOpenConns Maximun number of open connections
+  ## @param controller.persistence.nodeStatusOffLoad Save the entire workflow into etcd and DB
+  ## @param controller.persistence.archive Enable archiving old workflows
+  ## @param controller.persistence.postgresql.enabled Enable Postgresql persistence
+  ## @param controller.persistence.postgresql.host Host to use postgresql persistence
+  ## @param controller.persistence.postgresql.port Port for Postgresql
+  ## @param controller.persistence.postgresql.database Postgresql database
+  ## @param controller.persistence.postgresql.tableName Postgresql table name
+  ## @param controller.persistence.postgresql.userNameSecret.name Name of the secret containing the postgresql username
+  ## @param controller.persistence.postgresql.userNameSecret.key Key of the secret containing the postgresql username
+  ## @param controller.persistence.postgresql.passwordSecret.name Name of the secret containing the postgresql password
+  ## @param controller.persistence.postgresql.passwordSecret.key Key of the secret containing the postgresql password
+  ## @param controller.persistence.mysql.enabled Enable MySQL persistence
+  ## @param controller.persistence.mysql.host Host to use mysql persistence
+  ## @param controller.persistence.mysql.port Port for Postgresql
+  ## @param controller.persistence.mysql.database Postgresql database
+  ## @param controller.persistence.mysql.tableName Postgresql table name
+  ## @param controller.persistence.mysql.userNameSecret.name Name of the secret containing the mysql username
+  ## @param controller.persistence.mysql.userNameSecret.key Key of the secret containing the mysql username
+  ## @param controller.persistence.mysql.passwordSecret.name Name of the secret containing the mysql password
+  ## @param controller.persistence.mysql.passwordSecret.key Key of the secret containing the mysql password
+  ##
+  persistence:
+    enabled: false
+    connectionPool:
+      maxIdleConns: 100
+      maxOpenConns: 0
+    ## Save the entire workflow into etcd and DB
+    nodeStatusOffLoad: false
+    ## Enable archiving of old workflows
+    archive: false
+    postgresql:
+      enabled: false
+      host: '{{ include "argo-workflows.postgresql.fullname" . }}'
+      port: 5432
+      database: postgres
+      tableName: argo_workflows
+      ## the database secrets must be in the same namespace of the controller
+      userNameSecret:
+        name: argo-postgres-config
```

Comment:
Hardcoding this would be a bit problematic, because what about the case where the user specifies its own secret or it is using the `externalDatabase` section of the values (which, by the way, I think it is not here yet, right?)

Reply:
I added the dependencies in order to configure MySQL or postgresql automatically. That was not supported by upstream, and it is totally optional to use a database, so maybe what we should do is to only support external database  letting the user to configure all the parameters in that case.
I need to re-evaluate this after applying the changes to not provide config parameters as chart parameters.